### PR TITLE
List Doctor

### PR DESCRIPTION
**Pull request by kimshitong**
Closes 35

### CODE REVIEW

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -41,7 +46,9 @@ public class LogicManagerTest {
     public Path temporaryFolder;
 
     private Model model = new ModelManager();
+    private NewModel newModel = new NewModelManager();
```
Comment (yanghengtang) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:49]: @mounilsankar are we creating a NewLogicManagerTest or are we changing this one?

Comment (kimshitong) [reply src/test/java/seedu/address/logic/LogicManagerTest.java]: done

File: src/test/java/seedu/address/logic/NewLogicManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    public Path temporaryFolder;
+
+    private NewModel newModel = new NewModelManager();
+    private NewLogic newLogic;
+
+    @BeforeEach
+    public void setUp() {
+        JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(temporaryFolder.resolve("userPrefs.json"));
+        JsonDatabaseStorage databaseStorage =
+                new JsonDatabaseStorage(temporaryFolder.resolve("addressBook.json"));
```
Comment (yanghengtang) [inline src/test/java/seedu/address/logic/NewLogicManagerTest.java:48]: should be "database.json" instead of "addressBook.json"

Comment (kimshitong) [reply src/test/java/seedu/address/logic/NewLogicManagerTest.java]: done

**Review by yanghengtang — APPROVED** [review]
good to go

**Pull request merged**